src/gui/folderman.h

```diff
@@ -137,11 +137,11 @@ class FolderMan : public QObject
     bool startFromScratch(const QString &);
 
     /// Produce text for use in the tray tooltip
-    static QString trayTooltipStatusString(SyncResult::Status syncStatus, bool hasUnresolvedConflicts, bool paused);
+    static QString trayTooltipStatusString(SyncResult::Status syncStatus, bool hasUnresolvedConflicts, bool paused, const ProgressInfo *progress);
```

Comment:
depending if you mean a constant pointer or not, you most probably want to omit the `const` keyword in the declaration
with plain old data types, the fact that they are constant should be an implementation detail that is only made visible when implementing the method: no `const` in the declaration

all depends if you mean `const ProgressInfo *progress` or `ProgressInfo * const progress`

Reply:
ah, this was actually a leftover from when I still had `const ProgressInfo &` in there.

didn't know that the `const` could be omitted in the case of a constant pointer from the declaration, learned something new today :)